### ISSUE TITLE
file parse error!

### ISSUE THREAD

**Issue by YK1ngH** [liyucheng09/LLMs_Memorize#36]
File parse Error. ./results/qa/news_world-europe-65018434.tsv
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's attention_mask to obtain reliable results.
Setting pad_token_id to eos_token_id:128009 for open-end generation.
File parse Error. ./results/qa/news_uk-politics-65012967.tsv
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's attention_mask to obtain reliable results.
Setting pad_token_id to eos_token_id:128009 for open-end generation.
File parse Error. ./results/qa/news_uk-scotland-edinburgh-east-fife-65007151.tsv

when I run main.py,there are always errors like this, the tsv files in qa folder always have this question

example:

<img width="1658" height="962" alt="Image" src="https://github.com/user-attachments/assets/fda4c75d-3486-4cd4-b4b9-b60c5a878d55" />

 Read more: https://www.bbc.co.uk/news/uk-scotland-edinburgh-glasgow-60511155

---

Question 	Answer

What was the age of Romy Ulvestad when she died?  21

What was the reason for Romy's death?  She killed herself after staff failed to provide additional support, despite repeated warnings that she was struggling with her mental health.

What was the name of the campaign for the UK government to create legal duty of care for students?  The LEARN Network

How many signatures did the petition to the UK government reach before the deadline?  127,000

What was the reason for Libby Kitson joining the campaign?  She hopes the Scottish government will follow suit and create a legal duty of care for students.

What was the name of the university that Romy Ulvestad attended?  University of Edinburgh

What was the reason for the Department of Education saying that creating a legally enforced duty of care "would be a disproportionate response"?  They said it would be a disproportionate response.

What was the reason for Libby Kitson joining the campaign?  She hopes the Scottish government will follow suit and create a legal duty of care for students.

What was the name of the network that was set up by 25 bereaved families?  The LEARN Network

What was the reason for the Scottish government investing £11.5m to introduce additional counsellors in colleges and universities?  They are determined to support the mental health of all students.

What was the reason for the University of Edinburgh apologizing for failing Romy?  They had improved the support they offer since her death.

What was the reason for the University of Edinburgh improving the support they offer since Romy's death?  They had been "shocked and deeply saddened by what happened" to Romy and "deeply sorry" for the gaps in support given to her.

What was the reason for the University of Edinburgh saying that supporting their students' mental health and ensuring their wellbeing is their absolute priority?  They had been "shocked and deeply saddened by what happened" to Romy and "deeply sorry" for the gaps in support given to her.

What was the reason for the University of Edinburgh saying that their support policies and practices are under regular review?  They had been "shocked and deeply saddened by what happened" to Romy and "deeply sorry" for the gaps in support given
